envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/service/OAuthServerContainer.kt

```diff
@@ -0,0 +1,33 @@
+package pl.allegro.tech.servicemesh.envoycontrol.config.service
+
+import com.pszymczyk.consul.infrastructure.Ports
+import org.testcontainers.containers.Network
+import org.testcontainers.containers.wait.strategy.Wait
+import pl.allegro.tech.servicemesh.envoycontrol.config.testcontainers.GenericContainer
+
+class OAuthServerContainer :
+    GenericContainer<OAuthServerContainer>("kornelos/oauth-mock:0.0.2"),
```

Comment:
I am worried that the image might disappear one day. We should consider use some official or just build it.

Reply:
We talked with Lukasz and agreed that you could simply create a docker image "on the fly". See documentation: https://www.testcontainers.org/features/creating_images/ and serve a json over that.